truss-chains/examples/transcribe/whisper_model.py

```diff
@@ -36,29 +32,27 @@ class WhisperModel(chains.ChainletBase):
             ],
             external_package_dirs=[chains.make_abs_path_here(_LOCAL_WHISPER_LIB)],
         ),
-        compute=chains.Compute(gpu="A10G", predict_concurrency=128),
-        assets=chains.Assets(secret_keys=["hf_access_token"]),
+        compute=chains.Compute(gpu="H100", predict_concurrency=128),
     )
 
     def __init__(
         self,
-        context: chains.DeploymentContext = chains.depends_context(),
     ) -> None:
-        snapshot_download(
-            repo_id="baseten/whisper_trt_large-v3_A10G_i224_o512_bs8_bw5",
-            local_dir=context.data_dir,
-            allow_patterns=["**"],
-            token=context.secrets["hf_access_token"],
-        )
         from whisper_trt import WhisperModel
-
-        self._model = WhisperModel(str(context.data_dir), max_queue_time=0.050)
+        self._model = WhisperModel.from_model_name("large-v3", max_queue_time=0.010)
 
     async def run_remote(
```

Comment:
What do you think about separating the data to work on `audio_b64` from the parameters? I find this a useful categorical separation and it might make the other API changes better if these are not tied into a single pydantic model.